views/css/oobe.scss

```diff
@@ -0,0 +1,167 @@
+@import "web-os.css";
+@import "boot.css";
+@import "fonts.css";
+@import "mixins";
+@import "var";
+
+.base-pad {
+  padding-left: 4.5cm;
+}
+
+.web-os-back, html, body{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space